api/models/models.api.js

```diff
@@ -100,3 +100,24 @@ exports.postComment = (id, comment) => {
         });
     });
 };
+
+exports.patchVotes = (id, votes) => {
+  const articleIdNum = id.article_id;
+  const votesNum = votes.inc_votes;
+
+   if (typeof votesNum !== 'number' || isNaN(votesNum)) {
```

Comment:
We likely won't need to handle this with a custom error, see what happens when you pass the invalid query to psql. The table requires this to be a number so psql will likely throw an error for you which we could catch.